### ISSUE TITLE
CF netCDF standard_name updates for Group 1 and 2

### ISSUE THREAD

**Issue by ukmo-ansaulter** [NOAA-EMC/WW3#280]
A number of variables generated by the ww3_ounf netCDF processor do not have names documented in the CF conventions [standard_names table](https://cfconventions.org/standard-names.html) 

This is the first of several issues documenting proposal of standard_names and addition to ww3_ounf once ratified by CF. We are proposing names piecemeal so as to allow sensible discussions on each in the [CF forum](https://github.com/cf-convention/discuss/issues)

This batch covers input data in Group 1 and integrated parameters in Group 2 (submission on extreme wave parameters to follow). @mickaelaccensi , @ardhuin, @aliabdolali , @thesser1 , I'm making you aware of these now in case you have any immediate concerns or changes you'd like made to the proposals - although please note that the form of the names and description text follows existing CF convention style...

**Group 1**
META(1)%VARNS='air_sea_temperature_difference'
PROPOSE: 
Standard_name: temperature_difference_between_sea_water_and_air
Units: degree
Description: Sea water temperature is the in situ temperature of the sea water. Air temperature is the bulk temperature of the air, not the surface (skin) temperature.

META(1)%VARNS='icebergs_induced_attenuation_scale_for_waves'
PROPOSE:
Standard_name: sea_surface_wave_variance_spectral_density_attenuation_coefficient_due_to_icebergs
Units: None
Description: The attenuation coefficient is the fractional reduction of sea_surface_wave_variance_spectral_density due to the stated process. Attenuation is the sum of absorption and scattering.

META(1)%VARNS='sediment_grain_size'
PROPOSE:
Standard_name: sediment_grain_size
Units: m
Description: Average size of grains (also known as particles) in a sediment sample

META(1)%VARNS='maximum_ice_floe_diameter'
PROPOSE:
standard_name: sea_ice_maximum_floe_diameter      
Units: m
Description: Sea_ice means all ice floating in the sea which has formed from freezing sea water, rather than by other processes such as calving of land ice to form icebergs. An ice floe is a flat expanse of sea ice, generally taken to be less than 10 km across. Ice floe_diameter corresponds to the diameter of a circle with the same area as the ice floe.

**Group 2**
META(1)%VARNS='mean_wave_length' 
PROPOSE:
standard name: sea_surface_wave_mean_wavelength_from_variance_spectral_density_inverse_wavenumber_moment
units: m
description: The wave directional spectrum can be written as a five dimensional function S(t,x,y,k,theta) where t is time, x and y are horizontal coordinates (such as longitude and latitude), k is wavenumber and theta is direction. S has the standard name sea_surface_wave_directional_variance_spectral_density. S can be integrated over direction to give S1= integral(S dtheta) and this quantity has the standard name sea_surface_wave_variance_spectral_density. Wavenumber is the number of oscillations of a wave per unit distance. Wavenumber moments, M(n) of S1 can then be calculated as follows: M(n) = integral(S1 k^n dk), where k^n is k to the power of n. The inverse wave wavenumber, k(m-1), is calculated as the ratio M(-1)/M(0).

META(1)%VARNS='sea_surface_wave_peak_frequency'
PROPOSE:
standard_name: sea_surface_wave_frequency_at_variance_spectral_density_maximum
Units: s-1
Description: Frequency is the number of oscillations of a wave per unit time. The sea_surface_wave_ferquency_at_variance_spectral_density_maximum is the frequency of the most energetic waves in the total wave spectrum at a specific location.

META(1)%VARNS='sea_surface_wave_infragravity_significant_height' 
PROPOSE:
standard_name: sea_surface_infragravity_wave_significant_height
Units: m
Description: Significant wave height is a statistic computed from wave measurements and corresponds to the average height of the highest one third of the waves, where the height is defined as the vertical distance from a wave trough to the following wave crest. Infragravity waves are waves falling in the frequency range 0.04 to 0.004 s^-1 (wave periods of 25 to 250 seconds).

**Comment by ukmo-ansaulter** [comment]
Issues in this series are a follow on action to cf-convention/vocabularies#55

**Comment by ukmo-ansaulter** [comment]
Forcing data parameters in group 1 submitted to CF under [issue 89](https://github.com/cf-convention/vocabularies/issues/73)

**Comment by ukmo-ansaulter** [comment]
Integrated wave parameters in group 2 submitted to CF under [issue 90](https://github.com/cf-convention/vocabularies/issues/78)

**Comment by ukmo-ansaulter** [comment]
Proposals now added to the [cfeditor](http://cfeditor.ceda.ac.uk/proposals/1)

**Comment by ukmo-ccbunney** [comment]
@ukmo-ansaulter - Do you wish to keep this Issue open for future CF updates?

**Comment by ukmo-ansaulter** [comment]
Yes please - the intention is to add these to the WW3 defaults once CF update their [tables](https://cfconventions.org/standard-names.html) (which should be fairly soon, last update was in Jan 2021)
